examples/multi_file_bot/bot.py

```diff
@@ -0,0 +1,29 @@
+import asyncio
+import logging
+
+from aiogram import Bot, Dispatcher
+from aiogram.enums import ParseMode
+from examples.multi_file_bot import handlers
+
+# Bot token can be obtained via https://t.me/BotFather
+TOKEN = "42:TOKEN"
+
+
+async def main() -> None:
+    # Dispatcher is a root router
+    dp = Dispatcher()
+    # Register all the routers from handlers package (make sure they are imported into handlers/__init__.py)
```

Comment:
You are trying to optimize count of import lines but this is a bad way to be used in example, please use full import lines:

```python
from handlers.start import start_router
from handlers.echo import echo_router
```

and then use routers directly:
```python
start_router,
echo_router,
```

in this case is not needed to import things inside __init__ module